stormevents/nhc/track.py

```diff
@@ -981,55 +981,9 @@ def unfiltered_data(self, dataframe: DataFrame):
         # fill missing values of MRD and MSLP in the OFCL advisory
         if "OFCL" in self.advisories:
             tracks = separate_tracks(dataframe)
-
-            if "OFCL" in tracks:
-                ofcl_tracks = tracks["OFCL"]
-                carq_tracks = tracks["CARQ"]
-
-                for initial_time, forecast in ofcl_tracks.items():
-                    if initial_time in carq_tracks:
-                        carq_forecast = carq_tracks[initial_time]
-                    else:
-                        carq_forecast = carq_tracks[list(carq_tracks)[0]]
-
-                    relation = HollandBRelation()
-                    holland_b = relation.holland_b(
-                        max_sustained_wind_speed=carq_forecast[
-                            "max_sustained_wind_speed"
-                        ],
-                        background_pressure=carq_forecast["background_pressure"],
-                        central_pressure=carq_forecast["central_pressure"],
-                    )
-
-                    holland_b[holland_b == numpy.inf] = numpy.nan
-                    holland_b = numpy.nanmean(holland_b)
-
-                    mrd_missing = pandas.isna(forecast["radius_of_maximum_winds"])
-                    mslp_missing = pandas.isna(forecast["central_pressure"])
-                    radp_missing = pandas.isna(forecast["background_pressure"])
-
-                    # fill OFCL maximum wind radius with the first entry from the CARQ advisory
-                    forecast.loc[
-                        mrd_missing, "radius_of_maximum_winds"
-                    ] = carq_forecast["radius_of_maximum_winds"].iloc[0]
```

Comment:
The first `CARQ` track could be a `-24` hour "forecast", which has missing values for variables of interest. `iloc[0]` just gets the first entry of `carq_forecast` dataframe instead of getting the first hour-0 entry